{"uid":"c449921be97d0b1f","name":"test_more_articles_uploads","fullName":"Tests.Automata teszt.test_conduit.TestConduit#test_more_articles_uploads","historyId":"f5bb8eae81f0bbfba83f65ad86575797","time":{"start":1681477275230,"stop":1681477276020,"duration":790},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'hirek.csv'","statusTrace":"self = <test_conduit.TestConduit object at 0x7f936e13aa10>\n\n    def test_more_articles_uploads(self):\n        manipulatepages = model.ManipulatePages()\n    \n        getusers = model.GetUsers()\n    \n        print()\n        print('ATC 09 - Új adat bevitele')\n        print()\n    \n        # Bejelentkezem az alkalmazásba.\n        getusers.signin(self.browser, 'user1')\n    \n        # Megvárom, amíg megjelenik a \"Log out\" gomb az oldalon.\n        getusers.logout_btn(self.browser)\n    \n        # Feltöltök egy új cikket.\n>       article = manipulatepages.more_articles_uploads_from_data_source(self.browser)\n\nTests/Automata teszt/test_conduit.py:298: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nTests/Automata teszt/model.py:418: in more_articles_uploads_from_data_source\n    self.article_table_reading()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.ManipulatePages object at 0x7f936e1af100>\n\n    def article_table_reading(self):\n        article_list = []\n    \n>       with open('hirek.csv', 'r', encoding='UTF-8') as file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'hirek.csv'\n\nTests/Automata teszt/model.py:257: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestConduit","time":{"start":1681477272144,"stop":1681477275229,"duration":3085},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0}],"testStage":{"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'hirek.csv'","statusTrace":"self = <test_conduit.TestConduit object at 0x7f936e13aa10>\n\n    def test_more_articles_uploads(self):\n        manipulatepages = model.ManipulatePages()\n    \n        getusers = model.GetUsers()\n    \n        print()\n        print('ATC 09 - Új adat bevitele')\n        print()\n    \n        # Bejelentkezem az alkalmazásba.\n        getusers.signin(self.browser, 'user1')\n    \n        # Megvárom, amíg megjelenik a \"Log out\" gomb az oldalon.\n        getusers.logout_btn(self.browser)\n    \n        # Feltöltök egy új cikket.\n>       article = manipulatepages.more_articles_uploads_from_data_source(self.browser)\n\nTests/Automata teszt/test_conduit.py:298: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nTests/Automata teszt/model.py:418: in more_articles_uploads_from_data_source\n    self.article_table_reading()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.ManipulatePages object at 0x7f936e1af100>\n\n    def article_table_reading(self):\n        article_list = []\n    \n>       with open('hirek.csv', 'r', encoding='UTF-8') as file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'hirek.csv'\n\nTests/Automata teszt/model.py:257: FileNotFoundError","steps":[],"attachments":[{"uid":"b3b816449896e2e0","name":"stdout","source":"b3b816449896e2e0.txt","type":"text/plain","size":151}],"parameters":[],"shouldDisplayMessage":true,"hasContent":true,"stepsCount":0,"attachmentsCount":1},"afterStages":[{"name":"_xunit_setup_method_fixture_TestConduit::0","time":{"start":1681477276037,"stop":1681477276092,"duration":55},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0,"attachmentsCount":0}],"labels":[{"name":"parentSuite","value":"Tests.Automata teszt"},{"name":"suite","value":"test_conduit"},{"name":"subSuite","value":"TestConduit"},{"name":"host","value":"fv-az438-53"},{"name":"thread","value":"3047-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Tests.Automata teszt.test_conduit"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"c449921be97d0b1f.json","parameterValues":[]}